wren-ui/src/components/pages/setup/dataSources/TrinoProperties.tsx

```diff
@@ -0,0 +1,109 @@
+import { Form, Input, Switch } from 'antd';
+import { ERROR_TEXTS } from '@/utils/error';
+import { FORM_MODE } from '@/utils/enum';
+
+interface Props {
+  mode?: FORM_MODE;
+}
+
+export function TrinoProperties({ mode }: Props) {
+  const isEditMode = mode === FORM_MODE.EDIT;
+  return (
+    <>
+      <Form.Item
+        label="Display name"
+        name="displayName"
+        required
+        rules={[
+          {
+            required: true,
+            message: ERROR_TEXTS.CONNECTION.DISPLAY_NAME.REQUIRED,
+          },
+        ]}
+      >
+        <Input />
+      </Form.Item>
+      <Form.Item
+        label="Host"
+        name="host"
+        required
+        rules={[
+          {
+            required: true,
+            message: ERROR_TEXTS.CONNECTION.DISPLAY_NAME.REQUIRED,
+          },
+        ]}
+      >
+        <Input placeholder="10.1.1.1" disabled={isEditMode} />
+      </Form.Item>
+      <Form.Item
+        label="Port"
+        name="port"
+        required
+        rules={[
+          {
+            required: true,
+            message: ERROR_TEXTS.CONNECTION.DISPLAY_NAME.REQUIRED,
+          },
+        ]}
+      >
+        <Input disabled={isEditMode} />
+      </Form.Item>
+      <Form.Item
+        label="Catalog"
+        name="catalog"
+        required
+        rules={[
+          {
+            required: true,
+            message: ERROR_TEXTS.CONNECTION.DISPLAY_NAME.REQUIRED,
```

Comment:
please add a new one error dictionary to replace this. E.g., `ERROR_TEXTS.CONNECTION.CATALOG.REQUIRED`